--- ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/impl/AbstractTokenAntlrActionImpl.java.orig	2015-11-12 18:40:03.000000000 +0100
+++ ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/impl/AbstractTokenAntlrActionImpl.java	2015-11-12 19:22:10.392834532 +0100
@@ -29,6 +29,7 @@
  * </ul>
  *
  * @generated
+ * @since 3.0
  */
 public class AbstractTokenAntlrActionImpl extends AbstractElementImpl implements AbstractTokenAntlrAction {
 	/**
