--- ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/impl/GroupImpl.java.orig	2015-11-12 18:40:02.000000000 +0100
+++ ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/impl/GroupImpl.java	2015-11-12 19:38:48.828792775 +0100
@@ -40,6 +40,7 @@
 	 * @see #getGuardCondition()
 	 * @generated
 	 * @ordered
+	 * @since 3.0
 	 */
 	protected Condition guardCondition;
 	/**
@@ -65,6 +66,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public Condition getGuardCondition() {
 		return guardCondition;
@@ -74,6 +76,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public NotificationChain basicSetGuardCondition(Condition newGuardCondition, NotificationChain msgs) {
 		Condition oldGuardCondition = guardCondition;
@@ -89,6 +92,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public void setGuardCondition(Condition newGuardCondition) {
 		if (newGuardCondition != guardCondition) {
