--- ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/impl/XtextFactoryImpl.java.orig	2015-11-12 18:40:01.000000000 +0100
+++ ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/impl/XtextFactoryImpl.java	2015-11-12 19:37:52.116795147 +0100
@@ -384,6 +384,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public Condition createCondition() {
 		ConditionImpl condition = new ConditionImpl();
@@ -394,6 +395,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public Conjunction createConjunction() {
 		ConjunctionImpl conjunction = new ConjunctionImpl();
@@ -404,6 +406,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public Negation createNegation() {
 		NegationImpl negation = new NegationImpl();
@@ -414,6 +417,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public Disjunction createDisjunction() {
 		DisjunctionImpl disjunction = new DisjunctionImpl();
@@ -424,6 +428,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public CompositeCondition createCompositeCondition() {
 		CompositeConditionImpl compositeCondition = new CompositeConditionImpl();
@@ -434,6 +439,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public ParameterReference createParameterReference() {
 		ParameterReferenceImpl parameterReference = new ParameterReferenceImpl();
@@ -444,6 +450,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public LiteralCondition createLiteralCondition() {
 		LiteralConditionImpl literalCondition = new LiteralConditionImpl();
@@ -454,6 +461,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public InitAfterActions createInitAfterActions() {
 		InitAfterActionsImpl initAfterActions = new InitAfterActionsImpl();
@@ -464,6 +472,7 @@
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
 	 * @generated
+	 * @since 3.0
 	 */
 	public AbstractTokenAntlrAction createAbstractTokenAntlrAction() {
 		AbstractTokenAntlrActionImpl abstractTokenAntlrAction = new AbstractTokenAntlrActionImpl();
