--- ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/Conjunction.java.orig	2015-11-12 18:40:03.000000000 +0100
+++ ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/Conjunction.java	2015-11-12 19:43:14.017781684 +0100
@@ -12,6 +12,7 @@
  * @see org.eclipse.xtext.XtextPackage#getConjunction()
  * @model
  * @generated
+ * @since 3.0
  */
 public interface Conjunction extends CompositeCondition {
 } // Conjunction
