--- ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/ParserRule.java.orig	2015-11-12 18:40:02.000000000 +0100
+++ ./plugins/org.eclipse.xtext/emf-gen/org/eclipse/xtext/ParserRule.java	2015-11-12 19:09:52.958865374 +0100
@@ -156,6 +156,7 @@
 	 * @see org.eclipse.xtext.XtextPackage#getParserRule_InitAfterActions()
 	 * @model containment="true"
 	 * @generated
+	 * @since 3.0
 	 */
 	InitAfterActions getInitAfterActions();
 
@@ -166,6 +167,7 @@
 	 * @param value the new value of the '<em>Init After Actions</em>' containment reference.
 	 * @see #getInitAfterActions()
 	 * @generated
+	 * @since 3.0
 	 */
 	void setInitAfterActions(InitAfterActions value);
 
